graphql-schemas/schema.yml

```diff
@@ -3663,9 +3663,22 @@ confs:
   - { name: labels, type: json }
   - { name: name, type: string, isRequired: true }
   - { name: app, type: App_v1, isRequired: true }
-  - { name: namespaces, type: Namespace_v1, isList: true, isRequired: true }
+  - { name: namespaces, type: SLONamespaces_v1, isList: true, isRequired: true }
   - { name: slos, type: SLODocumentSLO_v1, isList: true }
 
+- name: SLONamespaces_v1
+  fields:
+  - { name: schema, type: string, isRequired: true }
+  - { name: path, type: string, isRequired: true }
+  - { name: namespace, type: Namespace_v1, isRequired: true}
+  - { name: prometheusAccess, type: SLOExternalPrometheusAccess_v1 }
+
+- name: SLOExternalPrometheusAccess_v1
+  fields:
+  - { name: schema, type: string, isRequired: true }
+  - { name: path, type: string, isRequired: true }
+  - { name: url, type: string, isRequired: true }
```

Comment:
`url` is required here but not in https://github.com/app-sre/qontract-schemas/pull/501/files#diff-25e1efebeeb9b86f33475e0ebbf223d3f568f35a72e4c6915e38e812f529a65eR33

Reply:
great spot, fixed!